package.json

```diff
@@ -17,6 +17,8 @@
     "release:run": "yarn build && yarn size && yarn publish --$RELEASE && yarn docs && yarn docs:deploy && yarn release:tag",
     "release:tag": "git push origin v$npm_package_version",
     "test": "jest",
+    "test:live": "LIVE=1 yarn test",
+    "test:live:win": "set LIVE=1&& yarn test",
```

Comment:
Just for my information, why is it needed besides `test:live`?

Reply:
`test:live` script doesn't work for windows, I introduced `test:live:win` )